lib/ruby_lsp/requests/support/common.rb

```diff
@@ -125,7 +124,7 @@ def markdown_from_index_entries(title, entries)
           categorized_markdown = categorized_markdown_from_index_entries(title, entries)
 
           <<~MARKDOWN.chomp
-            #{categorized_markdown[:title]}
+            #{"```ruby\n#{categorized_markdown[:code]}\n```"}
```

Comment:
I think this is fine, but the style is a little odd indeed. Can we do this instead?

```suggestion
            ```ruby
            #{categorized_markdown[:code]}
            ```
```